### ISSUE TITLE
Using LD estimates as input

### ISSUE THREAD

**Issue by apwilder** [esrud/GONE#38]
Hi Armando,
I'd like to try GONe on a low coverage dataset. I have LD estimates already generated using another program that uses genotype likelihoods for LD estimation from low coverage data. Is there a way to use those LD estimates as input for GONe? 
Thanks,
Aryn

**Comment by armando-caballero** [comment]
I do not think so, at least in a direct way. GONE works from the map/ped files. Sorry.
Best wishes,
Armando.
